src/main/java/org/quantumbadger/redreader/views/WebViewFixed.java

```diff
@@ -37,4 +44,14 @@ public void onWindowFocusChanged(final boolean hasWindowFocus) {
 			Log.e("WebView", "WebView.onWindowFocusChanged", ex);
 		}
 	}
+
+	private void setProxy(Context context) {
+		if(BaseActivity.getTorStatus()) {
+			try {
+				WebkitProxy.setProxy("org.quantumbadger.redreader.RedReader", context.getApplicationContext(), this, "127.0.0.1", 8118);
```

Comment:
This will break if the package name changes (e.g. in the Alpha version). Might be better to use `RedReader.getClass().getCanonicalName()`?